thirdparty/build-thirdparty.sh

```diff
@@ -1110,8 +1132,9 @@ build_brpc
 build_rocksdb
 build_librdkafka
 build_flatbuffers
-# must build before arrow
 build_jemalloc
+build_brotli
```

Comment:
brotli is built by arrow, we copied that after arrow is built, check the code line 621~624.

so we may not need to build it explicitly.